.github/dependabot.yaml

```diff
@@ -0,0 +1,28 @@
+---
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Hi @ljubon,

similarly to the [ParquetSharp.DataFrame](https://github.com/G-Research/ParquetSharp.DataFrame/pull/10#discussion_r1406806189), consuldotnet is a library for which we care about backwards compatibility. So I think it makes sense to leave the dependabot only for `github-actions`.